Information/CenterOfMass.FCMacro

```diff
@@ -795,23 +824,33 @@ class CenterofmassWidget(QtGui.QWidget):
             progress_bar.next()
         progress_bar.stop()
 
-    def compute_centerOfMass(self):
+    def compute_centerOfMass(self, setAllow=None):
         """Compute joint center of mass from all objects."""
```

Comment:
I have difficulties imagining what `setAllow`can mean. Would it be possible to rename the parameter or add an explanation in the DocString?